### PR TITLE
feat(sidebar): Updated empty state with annotations

### PR DESCRIPTION
**Pull request by ConradJChan**
TODO:
 -  [x] unit tests

Can annotate:
Screen Shot 2020-07-09 at 4 33 24 PM

Cannot annotate but can comment:
Screen Shot 2020-07-09 at 4 35 25 PM

Cannot annotate nor comment:
Screen Shot 2020-07-09 at 4 36 17 PM

### CODE REVIEW

File: i18n/en-US.properties [hunk 1]
```diff
@@ -373,7 +373,9 @@ be.nextFile = Next File
 # Title for next segment on skill timeline
 be.nextSegment = Next Segment
 # Message displayed in an empty activity feed
-be.noActivity = No Activity Yet
+be.noActivity = No activity to show
+# Message shown alongside empty activity feed when user can annotate
+be.noActivityAnnotationPrompt = Hover over the preview, and use the control at the bottom to annotate the file
```
Comment (jstoffan) [inline i18n/en-US.properties:378]: Can we remove the comma splice from this sentence (`, and`)? Should there be a period at the of the sentence to stay consistent with comments?

File: src/elements/content-sidebar/activity-feed/activity-feed/EmptyState.js [hunk 2]
```diff
@@ -6,24 +6,34 @@
...
+        : messages.noActivityCommentPrompt;
+
+    return (
+        <div className="bcs-activity-feed-empty-state">
+            {showAnnotationMessage ? <IconActivityFeedEmptyState /> : <IconActivityFeedEmptyStateFallback />}
+            <div className="bcs-empty-state-cta">
+                <FormattedMessage {...messages.noActivity}>
+                    {(text: string) => <span className="bcs-empty-state-detail">{text}</span>}
+                </FormattedMessage>
+                {showActionMessage ? (
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/EmptyState.js:29]: Can we switch to `&&` now that we're using a true boolean?

File: src/elements/content-sidebar/activity-feed/icons/IconActivityFeedEmptyState.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+import AccessibleSVG from '../../../../icons/accessible-svg';
+
+type Props = {
+    className?: string;
+    height?: number;
+    width?: number;
+};
+
+const IconActivityFeedEmptyState = ({ className = '', width = 140, height = 140 }: Props): JSX.Element => (
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/icons/IconActivityFeedEmptyState.tsx:11]: Is this considered to be an icon or an illustration? Seems like the latter.

File: src/elements/content-sidebar/activity-feed/icons/IconActivityFeedEmptyState.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <AccessibleSVG
+        className={`bcs-activity-feed-empty-state-illustration ${className}`}
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/icons/IconActivityFeedEmptyState.tsx:13]: Duplicated from the other illustration? Should this use SUIT?

File: src/elements/content-sidebar/activity-feed/icons/IconActivityFeedEmptyState.tsx (hunk 3 above)
Comment (ConradJChan) [reply src/elements/content-sidebar/activity-feed/icons/IconActivityFeedEmptyState.tsx:11]: Good point, will rename

File: i18n/en-US.properties [hunk 5]
```diff
@@ -373,7 +373,9 @@ be.nextFile = Next File
 # Title for next segment on skill timeline
 be.nextSegment = Next Segment
 # Message displayed in an empty activity feed
-be.noActivity = No Activity Yet
+be.noActivity = No activity to show
+# Message shown alongside empty activity feed when user can annotate
+be.noActivityAnnotationPrompt = Hover over the preview and use the control at the bottom to annotate the file.
```
Comment (jstoffan) [inline i18n/en-US.properties:378]: Should `control` be `controls`?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 6]
```diff
@@ -117,7 +117,12 @@
             }
         }
 
-        .bcs-activity-feed-empty-state-illustration {
+        .bcs-empty-state-detail {
+            font-weight: bold;
+        }
+
+        .bcs-ActivityFeedEmptyStateFallbackIllustration,
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:124]: May want to wrap the illustration in a container (maybe `.bcs-empty-state-image`) instead.

File: src/elements/content-sidebar/activity-feed/activity-feed/EmptyState.js [hunk 7]
```diff
@@ -6,24 +6,38 @@
...
-                </aside>
-            ) : null}
+const EmptyState = ({ showAnnotationMessage, showCommentMessage }: Props): React.Node => {
+    const showActionMessage = showAnnotationMessage || showCommentMessage;
+    const actionMessage = showAnnotationMessage
+        ? messages.noActivityAnnotationPrompt
+        : messages.noActivityCommentPrompt;
+
+    return (
+        <div className="bcs-activity-feed-empty-state">
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/EmptyState.js:23]: Worth rewriting the whole component to SUIT?

File: i18n/en-US.properties (hunk 5 above)
Comment (ConradJChan) [reply i18n/en-US.properties:378]: Good call, we'll be adding more in the future

File: src/elements/content-sidebar/activity-feed/activity-feed/EmptyState.scss [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    .bcs-EmptyState-cta {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        aside {
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/EmptyState.scss:11]: Ideally, this should use an non-nested class name.

Comment (ConradJChan) [reply src/elements/content-sidebar/activity-feed/activity-feed/EmptyState.scss:11]: Ah, good catch, let me fix that

**Pull request merged**